python/tvm/relay/op/contrib/cmsisnn.py

```diff
@@ -206,17 +206,17 @@ def qnn_avg_pool2d_pattern():
     def check_qnn_avg_pool2d(pattern):
         """Check if avg pool2d is supported by CMSIS-NN."""
         output = pattern
-        input_var = _find_last(pattern)
```

Comment:
We can remove the `_find_last` function now 😸

Reply:
ACK